analyzers/tests/SonarAnalyzer.UnitTest/TestCases/VariableUnused.cs

```diff
@@ -40,9 +40,6 @@ void F1()
             string c;
             c = "Hello, world!";
             Console.WriteLine(c);
-            
-            string f; // FN
```

Comment:
was the loss of ccov due to this removal?

Reply:
No, it's because for an unknown reason it picked up another file as well for the coverage computation.
Now it seems green but I see no coverage at all.